src/tree.rs

```diff
@@ -74,6 +74,9 @@ pub trait Element: Sized {
     /// Defaults to `is_link()`: when browsing history is not recorded, all links are "unvisited".
     fn is_unvisited_link(&self) -> bool { self.is_link() }
 
+    /// Returns whether this elements matches ':lang(lang)'
+    fn matches_lang(&self, lang: &Atom) -> bool;
```

Comment:
This is a breaking API change, please bump the version number in `Cargo.toml` to 0.3.0.

Should be all good with this!